torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -732,6 +734,14 @@ void AliasDb::analyzeSetAttr(Node* node) {
   registerWrite(self, node);
 }
 
+void AliasDb::analyzeSort(Node* node) {
```

Comment:
This is too specific to sort. This is not a sustainable way to add functionality.

Reply:
Well if you had a __lt__ function that wasn't pure, the schema for aten::sort([]t (a!) self) -> () wouldn't handle it